ledger/db/src/test_utils/mod.rs

```diff
@@ -310,7 +310,7 @@ pub fn initialize_ledger(
                 );
 
                 let key_images = tx.key_images();
-                (tx.prefix.outputs, key_images)
+                (tx.prefix.outputs.clone(), key_images)
```

Comment:
this was needed because rustc says we cannot move out of `Tx` anymore since it is a `Drop` type now